### ISSUE TITLE
Use PHP logic to set name in template

### ISSUE THREAD

**Issue by matthewrsj** [osuosl/invoiceplane-templates#2]
The name of the department, or `group`, creating the invoice should be written to the top left corner of the invoice, where the logo currently sits.

**Comment by matthewrsj** [comment]
This is the line that needs to be generalized: 
https://github.com/osuosl/invoiceplane-templates/blob/master/default-cass.php#L32

**Comment by matthewrsj** [comment]
A temporary solution is a custom field then just: 

```
<h1><?php echo $invoice->invoice_custom_your_osu_department_name_; ?></h1>
```

but users could put anything in there, even `banana`.

**Comment by matthewrsj** [comment]
Fixed by #7
